maps/map_files/USS_Almayer/USS_Almayer.dmm

```diff
@@ -140,8 +140,8 @@
 	name = "rat"
 	},
```

Comment:
Pipe under windows should be avoided
![bilde](https://user-images.githubusercontent.com/59782240/235793048-bf13b254-58df-4e20-8ace-1882a89ecc43.png)

Reply:
rest lgtm